samples/Common.c

```diff
@@ -549,10 +548,16 @@ STATUS createSampleStreamingSession(PSampleConfiguration pSampleConfiguration, P
 
     ATOMIC_STORE_BOOL(&pSampleStreamingSession->terminateFlag, FALSE);
     ATOMIC_STORE_BOOL(&pSampleStreamingSession->candidateGatheringDone, FALSE);
-
+    MEMSET(&pSampleStreamingSession->twccMetadata, 0x00, SIZEOF(TwccMetadata));
     pSampleStreamingSession->peerConnectionMetrics.peerConnectionStats.peerConnectionStartTime = GETTIME() / HUNDREDS_OF_NANOS_IN_A_MILLISECOND;
     // Flag to enable SDK to calculate selected ice server, local, remote and candidate pair stats.
     pSampleConfiguration->enableIceStats = FALSE;
+    pSampleConfiguration->disableTwcc = FALSE;
+
+    if (!pSampleConfiguration->disableTwcc) {
```

Comment:
Yes. Because we are setting to FALSE. But customer could change this to TRUE and the lock will get allocated for no reason. They might not remove the lock allocation.

Reply:
Can we enable through an env variable?
I am thinking it will make it faster to test what it is like with / without TWCC. Or building 2 executables one with enabled and disabled.